### ISSUE TITLE
Add missing aria-label to the input field on the Resources page

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#2122]
# Description

The input field on the [Resources page](https://www.if-me.org/resources) for filtering tags is missing an `aria-label`. Use the same value as the `placeholder` attribute. This will reduce the quality of the screen-reader experience.

![image](https://user-images.githubusercontent.com/3010728/193415629-d794a3c8-cdba-4567-94b7-b4f4644db985.png)

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by IvanSanchezSalvador** [comment]
Can i work on it?

**Comment by julianguyen** [comment]
Yeah go ahead, feel free to assign yourself to the ticket and after accepting the org invite. Thanks!
